src/pages/managerDashboard.js

```diff
@@ -42,12 +65,54 @@ const ManagerDashboard = () => {
         </>
       ) : (
         <>
-          <h1>Manager Dashboard</h1>
-          <button onClick={logout}>Logout</button>
+          {/* <button onClick={logout}>Logout</button> */}
+          <main className="bg-white container mx-auto  max-w-xs md:max-w-2xl lg:max-w-5xl">
+            <div className="bg-primary-lite my-10 pt-10 rounded-md flex flex-col lg:flex-row sm-text-center">
+              <div className="flex flex-col gap-5 p-10  lg:text-left">
+                <h1 className="text-5xl text-blue-800">
+                  Welcome Back, {state?.user?.displayName}
+                </h1>
+                <div className="mt-2">
+                  <Link
+                    href="#"
+                    className="text-slate-700  hover:text-blue-500 cursor-pointer text-2xl"
+                  >
+                    Edit Profile
```

Comment:
@istoga Do we have a UI for the "Edit Profile"? 👀

Reply:
@msarit We can open another PR and create a modal to pop up when Edit Profile is clicked on. I would love to merge this PR.